plugins/auth.js

```diff
@@ -35,6 +35,7 @@ export default function({ $loginEndpoint, store, $auth }) {
           store.commit('user/SET_USER_IS_ADMIN', response.is_admin)
           store.commit('user/SET_USER_CAN_ADMIN', response.can_admin)
           store.dispatch('artifacts/fetchFavorites', response.userid)
+          document.cookie = `session_id=${payload.token}`;
```

Comment:
We do not want to leak the token into another cookie.  If you want to use the token to generate a unique value, just hash it.

But then I'm curious why use the SSO token if you're using the Date for non-logged-in users, as above.

Reply:
My approach is to use the SSO token once the user logs in. But, initially, for all the users, I log in the date time. The cookie gets overwritten once the user logs in.

The SSO token is used as identification to link the views to the respective users. For non-logged in users (with the datetime cookie), we store the userid as null in the database.